src/ace/managers/viewport/tilebuffer.c

```diff
@@ -626,7 +648,9 @@ void tileBufferProcess(tTileBufferManager *pManager) {
 	pManager->ubStateIdx = !pManager->ubStateIdx;
 }
 
-void tileBufferRedrawAll(tTileBufferManager *pManager) {
+// This is used below with ubNonInterleavedBlit either 0 or 1, so with the inlining this is
+// a poor man's function template specialization...
+static inline void tileBufferRedrawAllInternal(tTileBufferManager *pManager, UBYTE ubNonInterleavedBlit) {
```

Comment:
I'm 95% sure that `inline` isn't reliable by itself, we should go with `__attribute__((always_inline))`. Bartman's debugger works fine with it btw.

Rename `ubNonInterleavedBlit` to `isInterleaved` and reverse its logic, not a fan of `isNotDoingSomething` or `isNotBeingSomething`, unless it's performance critical, e.g. skips cmp because of it being defaulted to zero in the fast path.

Reply:
Done